cyber/python/README.md

```diff
@@ -24,3 +23,6 @@ cyber.shutdown()
 Learn more usage from the [examples](cyber_py3/examples/) and
 [tests](cyber_py3/test/).
 
+**Note:** Like Cpp files, Python files are also managed by Bazel since Apollo 6.0. You can take
+[how_to_build_and_run_python_app.md](../../docs/howto/how_to_build_and_run_python_app.md)
```

Comment:
does docs/howto/how_to_build_and_run_python_app.md need to be  improved ?

Reply:
This document is a recent addition.